### PR TITLE
Des 85 dev make responsive path content

### PR DESCRIPTION
**Pull request by SthefaniaEcheverry**
Responsive view added for path content
Check details on:
(DES-85)[Responsive view for](https://danielcalleco.atlassian.net/jira/software/projects/DES/boards/4?selectedIssue=DES-85)

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]
Lgtm

File: src/App.css [hunk 1]
```diff
@@ -13,8 +13,9 @@
   --lightGrey: #ffffffb0;
   --darkGray: #707070;
   --greenDesafiante: #168991;
-  --yellow: #fad047;
+  --yellow: #fad047;
```
Comment (DanielCalleCO) [inline src/App.css:16]: did you run `pre-push` command BEFORE pushing the branch?

File: src/components/molecules/homeRoute/componentCenter/ComponentCenter.jsx [hunk 2]
```diff
@@ -2,25 +2,25 @@ import './componentCenter.css';
...
-              Identidad ¿quiénes somos <span> ¿De quiénes venimos? </span>{' '}
+          <div className="classRoomInfo-subcontent">
+            <p className="classRoomInfo-questions">
+              Identidad ¿quiénes somos?
             </p>
-            <p>
-              {' '}
-              ¡Conoce la historia de esta <br />
-              familia de la cuál tú eres parte!{' '}
+            <p className="classRoomInfo-questionsTwo">¿De quiénes venimos? </p>
```
Comment (DanielCalleCO) [inline src/components/molecules/homeRoute/componentCenter/ComponentCenter.jsx:15]: Is this space after 'venimo? ' necessary?

File: src/components/organims/home/homeViewFavorit/HomeViewFavorit.jsx [hunk 3]
```diff
@@ -1,11 +1,7 @@
-import { Sidebar } from '../../../molecules/Sidebar/Sibebar';
 import './homeViewFavorit.css';
 
 export const HomeViewFavorit = () => {
   return (
-    <>
-      <div className="container-homeViewFavorite">HomeViewFavorit</div>
-      <Sidebar />
-    </>
+    <div className="container-homeViewFavorite">HomeViewFavorit</div>
```
Comment (DanielCalleCO) [inline src/components/organims/home/homeViewFavorit/HomeViewFavorit.jsx:5]: Favorit**E**

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]
Please review the comments! You did it soooo good! Congrats!

**Review by Daniielmayo — APPROVED** [review]

**Review by Daniielmayo — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]
LGTM, nice job

**Pull request merged**